woocommerce/woocommerce-android
# Add shipping property (read-only)

Fixes #2519. This PR adds the shipping property to product variation detail screen.

![Screenshot_1592566609](https://user-images.githubusercontent.com/1522856/85130059-7f2e3680-b234-11ea-89a3-50dcf10ea11f.png)

**To test:**

1. Go to Products
2. Tap on a variable product that has shipping information
3. Notice the skeleton is shown while variation details are loading
4. After the the data is loaded, notice the shipping property is displayed with dimensions and correct units
5. Notice the shipping class is displayed (if the product has one assigned)
